src/Izzy/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Izzy Mansurov</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link href="../../dist/output.css" rel="stylesheet" />
+        <script src="../../dist/output.js"></script>
+        <script src="./navBar.js"></script>
+    </head>
+
+    <body x-data="navbar()" class="bg-slate-800">
+        <nav class="text-lg text-teal-400">
+            <div class="container mx-auto flex items-center justify-between px-4 py-3 lg:px-6">
+                <div class="w-1/3">
+                    <a href="#" class="font-fira text-lg hover:opacity-40">./izzyMansurov</a>
```

Comment:
```suggestion
                    <a href="#" class="font-mono text-lg hover:opacity-40">./izzyMansurov</a>
```
The correct name of the font is font-mono (in our case, `fira-code`). Check the tailwindcss.config.js file for more info about it.